docs/docusaurus.config.js

```diff
@@ -106,6 +106,11 @@ const config = {
         darkTheme: darkCodeTheme,
         additionalLanguages: ['cue', 'docker'],
       },
+      algolia: {
+        appId: 'KA4CNV2PUX',
+        apiKey: 'd65a6b6f6eb217f433716ba99aa51bc2',
```

Comment:
This seems bad? This is an open source repo.

Reply:
It's a public key.